src/App.js

```diff
@@ -1,7 +1,25 @@
-import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import { Login, Signup, Home } from "./pages/index";
+import { Navbar, RequiresAuth } from "./components/index";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Navbar />
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <RequiresAuth>
+              <Home />
+            </RequiresAuth>
+          }
+        />
+        <Route path="/login" element={<Login />} />
+        <Route path="/signup" element={<Signup />} />
+      </Routes>
```

Comment:
Instead of adding routes in the App.js file, you could move it to a separate file say AppRoutes.js and then add the component here. This way, in the future, if the number of routes increases your code will look cleaner, readable easy to understand.

Reply:
I will do that when done with all the features, good suggestion